src/CTA.WebForms2Blazor/ControlConverters/ButtonControlConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using HtmlAgilityPack;
+
+namespace CTA.WebForms2Blazor.ControlConverters
+{
+    public class ButtonControlConverter : ControlConverter
+    {
+        public ButtonControlConverter() : base()
```

Comment:
You don't need to make the call to base() if it doesn't do anything, if you just added it for use later though that's fine then